Description: ignore SSL tests (certificate keys too short)
Author: Xavier Guimard <yadd@debian.org>
Forwarded: no
Last-Update: 2020-12-15

--- a/test/test.js
+++ b/test/test.js
@@ -501,6 +501,7 @@
     });
   });
 
+  /*
   it("should follow redirects over https", function () {
     app.get("/a", redirectsTo("/b"));
     app.get("/b", redirectsTo("/c"));
@@ -517,6 +518,7 @@
         assert.deepEqual(res.responseUrl, "https://localhost:3601/c");
       });
   });
+  */
 
   it("should destroy responses", function () {
     app.get("/a", hangingRedirectTo("/b"));
@@ -858,6 +860,7 @@
   });
 
   describe("should handle cross protocol redirects ", function () {
+    /*
     it("(https -> http -> https)", function () {
       app.get("/a", redirectsTo("http://localhost:3600/b"));
       app2.get("/b", redirectsTo("https://localhost:3601/c"));
@@ -891,6 +894,7 @@
           assert.deepEqual(res.responseUrl, "http://localhost:3600/c");
         });
     });
+    */
   });
 
   describe("should error on an unsupported protocol redirect", function () {
@@ -1523,6 +1527,7 @@
     });
   });
 
+  /*
   describe("should choose the right agent per protocol", function () {
     it("(https -> http -> https)", function () {
       app.get("/a", redirectsTo("http://localhost:3600/b"));
@@ -1556,6 +1561,7 @@
         });
     });
   });
+  */
 
   describe("should not hang on empty writes", function () {
     it("when data is the empty string without encoding", function () {
